app/assets/javascripts/libfigure/define-figure.js

```diff
@@ -176,6 +176,15 @@ function aliases(move) {
 }
 
 function moves() {
+ return Object.keys(defined_events).sort(function(a,b) {
+   var aa = a.toLowerCase();
+   var bb = b.toLowerCase();
+   if (aa < bb) { return -1 ;}
+   else if (aa > bb) { return 1; }
+   else { return 0; }
+}
+
+function movesMenuOrdering() {
  var m = Object.keys(defined_events).sort(function(a,b) {
    var aa = a.toLowerCase();
```

Comment:
Proposal: this function duplicates a lot of code from `moves`. DRY (Don't Repeat Yourself) it up by having it initialize `m` to be the output of `moves()`, rather than duplicating that hairy invocation of `Object.keys(...).sort(...)`

Reply:
doh.  that makes sense.  on it.